### PR TITLE
Fixed skybox, visible, and interactable components for spoke

### PR DESCRIPTION
**Pull request by robertlong**
- Set the skybox scale to the distance property (Note this is a breaking change. Environments will need the skybox node's scale set to 1, 1, 1)
- Adds the interactable component which just adds the interactable css-class
- All spoke components must be objects. The visible component inflator now also accepts an object

### CODE REVIEW

File: src/gltf-component-mappings.js [hunk 1]
```diff
@@ -4,6 +4,9 @@ AFRAME.GLTFModelPlus.registerComponent("quack", "quack");
 AFRAME.GLTFModelPlus.registerComponent("sound", "sound");
 AFRAME.GLTFModelPlus.registerComponent("collision-filter", "collision-filter");
 AFRAME.GLTFModelPlus.registerComponent("css-class", "css-class");
+AFRAME.GLTFModelPlus.registerComponent("interactable", "css-class", (el, componentName) => {
```
Comment (netpro2k) [inline src/gltf-component-mappings.js]: Why not actually just use the css class component?

File: src/gltf-component-mappings.js [hunk 2]
```diff
@@ -43,7 +46,13 @@ AFRAME.GLTFModelPlus.registerComponent(
     };
   })()
 );
-AFRAME.GLTFModelPlus.registerComponent("visible", "visible");
+AFRAME.GLTFModelPlus.registerComponent("visible", "visible", (el, componentName, componentData) => {
```
Comment (johnshaughnessy) [inline src/gltf-component-mappings.js]: How come the `visible` component needs this custom `inflator`?

File: src/gltf-component-mappings.js (hunk 1 above)
Comment (brianpeiris) [reply src/gltf-component-mappings.js]: Spoke cannot export components with literal prop values. Prop values are always objects.

File: src/gltf-component-mappings.js (hunk 2 above)
Comment (brianpeiris) [reply src/gltf-component-mappings.js]: Same as above, Spoke cannot export components with literal prop values. Prop values are always objects so we need to use a custom inflator to translate the value to a-frame but also maintain backwards compat for our existing environments (which we will eventually convert to the new format).

**Pull request merged**